Godeps/Godeps.json

```diff
@@ -105,11 +105,11 @@
 		},
 		{
 			"ImportPath": "github.com/apache/incubator-openwhisk-client-go/whisk",
-			"Rev": "b3d194d8b937a801ee11efcf7f2544d9cea1f58e"
+			"Rev": "8a672e5f39dd0bbdf6ca9460782557d7f449103c"
```

Comment:
@houshengbo is there a way to automate this?  
AND is there a way for 'wskdeploy' to somehow say built on "Go Client version X"?

Reply:
I have no idea about how to automate the version pinning. have to investigate more.